output/index.html

```diff
@@ -24,21 +24,159 @@
+                        <span class="job">Back End Dev</span><i class="fa-solid fa-circle"></i>
+                        <span class="year">2015</span>
+                    </div>
+                    <div class="desc">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </div>
+                    <ul class="tags">
+                        <li class="tag html">html</li> 
+                        <li class="tag css">css</li><li class="tag javascript">javaScript</li>
+                    </ul>
+                    <div class="button">
+                        See Project
+                    </div>
+                </div>
+            </section>
+          
+            <section class="cardwork">
+                <div class="snapshot4">
+                </div>
+                <div class="info">
+                    <h1 class="cardwork-title">Multi-Post Stories</h1>
+                    <div class="subtitle">
+                        <span class="company">CANOPY</span><i class="fa-solid fa-circle"></i>
+                        <span class="job">Back End Dev</span><i class="fa-solid fa-circle"></i>
+                        <span class="year">2015</span>
+                    </div>
+                    <div class="desc">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </div>
+                    <ul class="tags">
+                        <li class="tag html">html</li> 
+                        <li class="tag css">css</li>
+                        <li class="tag javascript">javaScript</li>
+                    </ul>
+                    <div class="button">
+                        See Project
+                    </div>
+                </div>
+            </section>
+            </div>
+
+            <div class="space">
+                
+            </div>
+            <section class="about">
+                <h1 class="about-title">About Myself</h1>
+                <p class="about-desc">Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                <div class="handle">
+                    <div class="connect">LET'S CONNECT</div>
+                    <ul class="social-media">
+                        <li class="media-item"><i class="fa-brands fa-twitter"></i></li>
+                        <li class="media-item"><i class="fa-brands fa-linkedin-in"></i></li>
+                        <li class="media-item"><i class="fa-brands fa-github"></i></li>
+                        <li class="media-item"><i class="fa-brands fa-facebook"></i></li>
+                    </ul>
+                </div>
+                <div class="about-button">
+                    Get my resume
+                </div>
```

Comment:
- [ ] Kindly adjust this button, so it sticks to the design :+1: 
Your Design

![Screenshot from 2023-01-13 16-56-25](https://user-images.githubusercontent.com/80458199/212350328-2c4bf5c7-31e1-4107-939b-97df7079f9b2.png)

Figma Design

![Screenshot from 2023-01-13 16-58-01](https://user-images.githubusercontent.com/80458199/212350486-24dca433-5e89-48e2-830e-34d85ceb40b1.png)